docs/changelog.md

```diff
@@ -1,4 +1,9 @@
 # TBD
+### Features
```

Comment:
Quick note on all these - I tend to like putting the name of the class (e.g. `LambdaConfigurator`) in the changelog rather than the filename (`lambda_configurator`), because class names seem to be pretty constant across languages - Go, Java, and Javascript all do UpperCamelCase for class whereas filename differs by language

Reply:
yeap! it make senses